src/Sylius/Bundle/CoreBundle/Resource/StateMachine/CompositeStateMachine.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Resource\StateMachine;
```

Comment:
In my practice of extension sylius components/bundles, following original naming scheme strictly is the best course of action.

It's a bit controversial to have `src\Resource\Controller\StateMachine`, but that's how `sylius\resource` does things and should be alright.

```suggestion
namespace Sylius\Bundle\CoreBundle\Resource\Controller\StateMachine;
```